modules/apps/object/object-service/service.xml

```diff
@@ -1190,6 +1190,7 @@
 		<exception>LockedObjectAction</exception>
 		<exception>ObjectActionActive</exception>
 		<exception>ObjectActionConditionExpression</exception>
+		<exception>ObjectActionDescriptionExpression</exception>
```

Comment:
@Aquiiiles it should be `<exception>ObjectActionDescription</exception>`.

And you should always run `gw buildService` after changing this file..

See https://github.com/brianchandotcom/liferay-portal/pull/155043:

1. https://github.com/brianchandotcom/liferay-portal/pull/155043/commits/8d1756c0a73a9080a48f3281eafc7f910d85a291
2. https://github.com/brianchandotcom/liferay-portal/pull/155043/commits/dafd488ebb63465a684d8db788d88fa361957cb5

Reply:
The `gw buildService` command will generate the `ObjectActionDescriptionException` class and after that you can apply your changes in this class.